README.md

```diff
@@ -144,3 +147,329 @@ Otherwise you will need to tell the VM each time as well like so:
 ```
 java -Djava.library.path=wherever-i-downloaded-the-jnilib -jar hellbender-protected.jar ...
 ```
+
+The CNV case and PoN workflows (description and examples)
+---------------------------------------------------------
+
+#### Requirements
+1. Java 1.8
+2. A functioning GATK4-protected jar (hellbender-protected.jar)
+3. HDF5 1.8.13 
+4. The location of the HDF5-Java JNI Libraries Release 2.9 (2.11 for Macs).  (See platform instructions above for typical locations)  
+5. Reference genome (fasta) with fai and dict indexes.  This can be downloaded as part of the GATK resource bundle: http://www.broadinstitute.org/gatk/guide/article?id=1213
+6. target BED file that was used to create the PoN file.  Format details can be found `here <http://genome.ucsc.edu/FAQ/FAQformat.html#format1>`_ .  **NOTE:**  For the CNV tools, you will need a fourth column for target name, which must be unique across rows.
+```
+1       12200   12275   target1
+1       13505   13600   target2
+1       31000   31500   target3
+1       35138   35174   target4
+....snip....
+```
+7. PoN file (when running case samples only).  This file should be created using the Create PoN workflow (see below).
+8. Sample name (when running case samples only).  This can be extracted from the input bam file by looking at the ``SM`` tag.  Use ``samtools view -H my_sample.bam | egrep SM``.
+
+#### Case sample workflow
+
+This workflow requires a PoN file generated by the Create PoN workflow.
+
+If you do not have a PoN, please skip to the [Create PoN workflow](#create-pon-workflow), below ....
+
+##### Overview of steps
+
+- Step 1. Collect proportional coverage
+- Step 2. Create coverage profile
+- Step 3. Segment coverage profile
+- Step 4. Call segments
+
+##### Step 1. Collect proportional coverage
+
+###### Inputs
+- bam file
+- target bed file -- must be the same that was used for the PoN
+- reference_sequence (required by GATK) -- fasta file with hg19 reference.  
```

Comment:
@vdauwera I need a little guidance here....  We use:
``REF=/seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta``
Should we still be renaming everything b37?  Or should we change the reference fasta we use?